deploy/crds/operator.ibm.com_v1alpha1_metaoperatorset_cr.yaml

```diff
@@ -4,6 +4,9 @@ metadata:
   name: common-service
 spec:
   services:
+  - name: ibm-metering-operator
+    state: present
```

Comment:
please set it to absent, later we will do a bulk update on this flag.

Reply:
done